functional-tests/src/test/resources/GebConfig.groovy

```diff
@@ -95,7 +95,7 @@ environments {
 			ChromeOptions o = new ChromeOptions()
 			o.addArguments('headless')
 			//o.addArguments('disable-gpu') 
-			//o.addArguments('no-sandbox')
+			//o.addArguments('no-tools')
```

Comment:
This should be sandbox. It is a chrome flag for the browser testing suite

Reply:
Oh yeah, Thanks. Added that back